02-peer-protocol.md

```diff
@@ -453,6 +453,9 @@ completed.
     2. data:
         * [`s64`:`satoshis`]
    1. type: 2 (`require_confirmed_inputs`)
+   1. type: 5 (`request_funding`)
```

Comment:
nit: If we want an odd type here, why not `3`? Doesn't seem claimed as far as I can see in other PRs, unless I'm missing one?

Reply:
Ah, sorry.